cmake/protobuf-config.cmake.in

```diff
@@ -40,6 +40,8 @@ function(protobuf_generate)
 
   if(protobuf_generate_EXPORT_MACRO AND protobuf_generate_LANGUAGE STREQUAL cpp)
```

Comment:
If `EXPORT_MACRO` is set and we're generating C++, shouldn't we still adhere to `PLUGIN_OPTIONS`? It seems to me like in that case we just need to pass both `dllexport_decl=` and `PLUGIN_OPTIONS`, separated by a comma.

Reply:
Did not know one can just pass them comma separated.
What about this?
Let me try a bit more testing e.g. with the grpc plugin.

Reply:
I verified this works:
```
protobuf_generate(
  TARGET
  GeneratedProtos
  PLUGIN_OPTIONS
  lite
  PLUGIN_OPTIONS
  lite
  PROTOC_OUT_DIR
  ${CMAKE_BINARY_DIR}
  IMPORT_DIRS
  ${CMAKE_CURRENT_SOURCE_DIR})

protobuf_generate(
  TARGET
  GeneratedProtos
  PLUGIN_OPTIONS
  generate_mock_code=true
  LANGUAGE
  grpc
  PROTOC_OUT_DIR
  ${CMAKE_BINARY_DIR}
  GENERATE_EXTENSIONS
  .grpc.pb.h
  .grpc.pb.cc
  PLUGIN
  protoc-gen-grpc=$<TARGET_FILE:gRPC::grpc_cpp_plugin>
  IMPORT_DIRS
  ${CMAKE_CURRENT_SOURCE_DIR})
```